MVVM_Movies/MVVM_Movies/ViewController/MoviesModule/ViewModel/MoviesViewModel.swift

```diff
@@ -0,0 +1,71 @@
+// MoviesViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
+/// Протокол Мувис вьюмодели
+protocol MoviesViewModelProtocol {
+    var movies: [ResultsFilm] { get set }
+    var moviesTableViewReloadData: (() -> ())? { get set }
+    func getMovies()
+    func getImage(url: URL, complition: @escaping (UIImage) -> Void)
+}
+
+/// MoviesViewModel
+final class MoviesViewModel: MoviesViewModelProtocol {
+    var funcprint: (() -> ())?
```

Comment:
```suggestion
    var funcprint: (() -> ())?
```
что за название funcprint?